src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -490,6 +506,7 @@ ${chalk.dim("In case we're mistaken, please report this to us 🙏.")}`)
     return printGeneratorConfig(fixedGenerator)
   }
 
+  // TODO ??
```

Comment:
Huh?

Reply:
Naming is confusing since it doesn't print, it's used for this 
https://github.com/prisma/prisma/blob/eeca05f1d460abd81012c0bbc974d2d71ac6d921/src/packages/engine-core/src/NodeEngine.ts#L532

Reply:
There I am equally confused what it really does to be honest :/

Reply:
https://github.com/prisma/prisma/pull/7730#discussion_r657089170

Reply:
```suggestion
  // TODO give better name (see usages)
```